fe/fe-core/src/main/java/com/starrocks/catalog/TableFunctionTable.java

```diff
@@ -93,6 +93,7 @@ public class TableFunctionTable extends Table {
     public static final String PROPERTY_CSV_ENCLOSE = "csv.enclose";
     public static final String PROPERTY_CSV_ESCAPE = "csv.escape";
     public static final String PROPERTY_CSV_TRIM_SPACE = "csv.trim_space";
+    public static final String PROPERTY_STRICT_MDDE = "strict_mode";
```

Comment:
I think it's better to use a single STRICT_MODE const variable.
For example, it can be put in com.starrocks.load.LoadProperties